FizzBazz.js

```diff
@@ -1,13 +1,11 @@
-const fizzBuzz = () => {
-    console.log('text');
+function fizzBuzz() {
+     /* add alternative solution (ternary operator) */
```

Comment:
No need in comment anymore, I think we will leave this solution

Reply:
OK